sdk/maps/Azure.Maps.Search/README.md

```diff
@@ -0,0 +1,103 @@
+# Azure Maps Search client library for .NET
+
+Azure Maps Search is a library that can query for locations, points of interests or search within a geometric area.
+
+[Source code](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/maps/Azure.Maps.Search/src) | [API reference documentation](https://docs.microsoft.com/rest/api/maps/) | [REST API reference documentation](https://docs.microsoft.com/rest/api/maps/search) | [Product documentation](https://docs.microsoft.com/azure/azure-maps/)
+
+## Getting started
+
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/ ):
+
+```dotnetcli
+dotnet add package Azure.Maps.Search --prerelease
+```
+
+### Prerequisites
+
+> You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/) and [Azure Maps account](https://docs.microsoft.com/azure/azure-maps/quick-demo-map-app#create-an-azure-maps-account).
+
+To create a new Azure Maps account, you can use the Azure Portal, Azure PowerShell, or the Azure CLI. Here's an example using the Azure CLI:
+
+```powershell
+az maps account create --kind "Gen2" --disable-local-auth true --account-name "myMapAccountName" --resource-group "<resource group>" --sku "G2" --accept-tos
+```
+
+### Authenticate the client
+
+There are 2 ways to authenticate the client: Shared key authentication and Azure AD.
+
+#### Shared Key Authentication
+
+* Go to Azure Maps account > Authentication tab
+* Copy `Primary Key` or `Secondary Key` under **Shared Key Authentication** section
+
+```C# Snippet:InstantiateSearchClientViaSubscriptionKey
+// Create a SearchClient that will authenticate through Subscription Key (Shared key)
+var credential = new AzureKeyCredential("<My Subscription Key>");
+SearchClient client = new SearchClient(credential);
+```
+
+#### Azure AD Authentication
+
+In order to interact with the Azure Blobs Storage service, you'll need to create an instance of the BlobServiceClient class. The [Azure Identity library](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/identity/Azure.Identity/README.md) makes it easy to add Azure Active Directory support for authenticating Azure SDK clients with their corresponding Azure services.
```

Comment:
Change the first sentence to Azure Maps Search service. Please copy the statement from [Render service](https://github.com/dubiety/azure-sdk-for-net/tree/feature/maps-render/sdk/maps/Azure.Maps.Render) description